tests/RobotFramework/tests/tedge_connect/tedge_connet_mqtt_service.robot

```diff
@@ -0,0 +1,53 @@
+*** Settings ***
+Resource            ../../resources/common.resource
+Library             DateTime
+Library             Cumulocity
+Library             ThinEdgeIO
+
+Test Teardown       Get Logs
+
+
+*** Test Cases ***
+Connect to Cumulocity MQTT Service endpoint
+    ${DEVICE_SN}=    Setup    connect=${False}
+    Execute Command    tedge config set c8y.mqtt_service.enabled true
+    Execute Command    tedge config set c8y.mqtt_service.topics 'sub/topic,demo/topic'
+    Execute Command    tedge connect c8y
+
+    # TODO: Subscribing to test/topic from another client
+    Execute Command    tedge mqtt pub c8y-mqtt/test/topic '"hello"'
+    # TODO: Validate message received on test/topic on the other client
```

Comment:
We really need to find a way to test a connection the MQTT service. Exchanging messages from one client to another will not work.